tests/acceptance/features/apiSpaces/shareSubItemOfSpace.feature

```diff
@@ -62,3 +62,15 @@ Feature: Share a file or folder that is inside a space
       | file.txt | editor    | 404        | No share permission |
       | file.txt | viewer    | 404        | No share permission |
       | folder   | viewer    | 404        | No share permission |
+
+
+  Scenario Outline: An user participant of the project space can see the created resources share
+    Given user "Alice" has shared a space "share sub-item" to user "Brian" with role "<spaceRole>"
+    When user "Alice" shares the following entity "file.txt" inside of space "share sub-item" with user "Bob" with role "editor"
```

Comment:
Status check after the request would be appropriate.

Reply:
The scenario is not to create a link share but to check that space members can see the created share so that Victor might have written this scenario this way.

Any thoughts?
CC @individual-it

Reply:
I think we don't need the status check, because that is check in the tests above